packages/terra-form-select/src/shared/_OptGroup.jsx

```diff
@@ -39,7 +39,7 @@ const OptGroup = ({
 }) => {
   const theme = React.useContext(ThemeContext);
   return (
-    <li className={cx('opt-group', { 'is-disabled': disabled }, theme.className)} role="group">
+    <li className={cx('opt-group', { 'is-disabled': disabled }, theme.className)} role="option">
```

Comment:
I'm a little suspicious about this change. Can you post the error message that was being thrown?

Reply:
>  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": [
            "option"
          ],
          "id": "aria-required-children",
          "impact": "critical",
          "message": "Required ARIA child role not present: option",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  Required ARIA child role not present: option",
      "html": "<ul id=\"terra-select-menu-25\" role=\"listbox\" class=\"_Menu-module__menu___33lDo\" aria-label=\"Menu\" aria-activedescendant=\"terra-select-option-green\" tabindex=\"0\">",
      "impact": "critical",
      "none": [],
      "target": [
        "#terra-select-menu-25"
      ]
    }

Was the original error however I am no longer getting that and am instead getting 

>  "nodes": [
    {
      "all": [],
      "any": [
        {
          "data": [
            "aria-selected=\"false\""
          ],
          "id": "aria-allowed-attr",
          "impact": "critical",
          "message": "ARIA attribute is not allowed: aria-selected=\"false\"",
          "relatedNodes": []
        }
      ],
      "failureSummary": "Fix any of the following:\n  ARIA attribute is not allowed: aria-selected=\"false\"",
      "html": "<li class=\"_OptGroup-module__opt-group___hROs6\" role=\"group\" aria-selected=\"false\">",
      "impact": "critical",
      "none": [],
      "target": [
        "._OptGroup-module__opt-group___hROs6[role=\"group\"]:nth-child(1)"
      ]
    }

I'm going to see if i just remove aria-selected off the OptGroup `<li>` fixes the issue

Reply:
Okay so If I remove `aria-selected="false"` from the `<li>` I get the original top error.